dashboard/src/pages/packages/index.vue

```diff
@@ -103,19 +108,60 @@ const toggleLegend = () => {
         </tbody>
       </table>
     </div>
-    <div>
-      <nav aria-label="Package list pages">
+    <div v-if="packageStore.pager.total > 1">
+      <nav role="navigation" aria-label="Pagination navigation">
         <ul class="pagination justify-content-center">
+          <li v-if="packageStore.pager.total > packageStore.pager.maxPages">
+            <a
+              href="#"
+              :class="{
+                'page-link': true,
+                disabled: packageStore.pager.current == 1,
+              }"
+              aria-label="Go to first page"
+              @click.prevent="packageStore.fetchPackages(1)"
+            >
+              <IconSkipStartFill /> First
```

Comment:
The first and last links seem unnecessary when you have all the pages being displayed.

Reply:
The first and last buttons aren't displayed if all the pages are displayed in the pager.  Here's an example with 200 packages (20 per page):
![image](https://github.com/user-attachments/assets/9b6ee67d-793e-4903-99fa-13537225403b)